content/03.methods.md

```diff
@@ -14,9 +14,36 @@ talk about generative model and how it works
 ### Word Embeddings
 mention facebooks fasttext model and how we used it to train word vectors
 ### Discriminator Model
-talk about the discriminator model and how it works
+The discriminator model is designed to improve off of the generated labels from the generative model.
+We used a convolutional neural network with multiple filters as our discriminator model.
+This network uses multiple filters with fixed widths of 300 dimensions and can have varying heights (Figure {@fig:convolutional_network}).
+For our model we each filter at a fixed height of 7, because this provided the best performance in terms of relationship classification [@arxiv:1510.03820].
+We trained this model for 20 epochs using the adam optimizer [@arxiv:1412.6980] with a learning rate of 0.001.
+This optimizer had beta values of 0.9 and 0.99 respectively.
```

Comment:
Why respectively? I might have missed that there are two things there.

Reply:
The adam optimizer takes two values for beta. According to the [pytorch documentation](https://pytorch.org/docs/stable/optim.html#torch.optim.Adam):
> **beta** (Tuple[float, float], optional) - coefficients used for computing running averages of gradient and its square

Turns out the values I'm using are the default, so I can change this to say default settings if too confusing.

Reply:
Yes - I think that would be helpful